core-bundle/tests/EventListener/GlobalsMapListenerTest.php

```diff
@@ -36,25 +36,25 @@ public static function getValuesData(): iterable
     {
         yield 'add single' => [
             [],
-            [0 => ['text' => 'HeadlineFragment']],
+            [['text' => 'HeadlineFragment']],
```

Comment:
I don't think this change is a good idea, because `0` reflects the designated priority. Obviously the values are the same, but it should show them in the tests.

Reply:
Ah, sorry, I only saw your comment now. I will revert this, although PhpStorm complains about it, too:

<img width="483" alt="" src="https://github.com/user-attachments/assets/5c85c572-06e5-42e7-a52c-cc5365b84095" />